src/slo/x_days_sli/sli_results_streamer.py

```diff
@@ -1,12 +1,22 @@
 import logging
 
+from src.commons.big_query.streaming.data_streamer import DataStreamer
+from src.commons.config.configuration import configuration
 
-class SLIResultsStreamer(object):
 
-    def __init__(self, big_query):
-        self.big_query = big_query
+class SLIResultsStreamer(object):
+    def __init__(self,
+                 project_id=configuration.backup_project_id,
+                 dataset_id="SLO_views_legacy",
```

Comment:
It streams result to partitioned table, so it shouldn't go to dataset with 'views' in name.

Reply:
Let's ignore it now :)

Reply:
I would move the table to the other dataset as well, then we could have different ACLs for them, i.e. read-only for table and read/write for views